test/module/irohad/validation/chain_validation_test.cpp

```diff
@@ -74,7 +55,8 @@ class ChainValidationTest : public ::testing::Test {
  */
 TEST_F(ChainValidationTest, ValidCase) {
   // Valid previous hash, has supermajority, correct peers subset => valid
-  auto block = getBlockBuilder().build();
+  BlockMock block;
```

Comment:
Can we make it a pointer here as below? Then we can move it to a class member and call setupBlock() in SetUp().